### PR TITLE
Challenge/anna sala/to do list redux

### PR DESCRIPTION
**Pull request by annasm07**

### CODE REVIEW

File: anna-sala/challenges/to-do-list-redux/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
```
Comment (gteixi) [inline anna-sala/challenges/to-do-list-redux/src/index.jsx:3]: els fitxers del css son els ultims a importar!

File: anna-sala/challenges/to-do-list-redux/src/redux/actions/actionCreators.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import actionTypes from './actionTypes';
+
+export function loadTasks() {
+  return {
+    type: actionTypes.LOAD_TASKS,
```
Comment (gteixi) [inline anna-sala/challenges/to-do-list-redux/src/redux/actions/actionCreators.js:5]: aqui no hauries de fer un return de la array TASKS?

**Review by gteixi — APPROVED** [review]

**Review by edalandete — APPROVED** [review]

**Review by jasuarez21 — APPROVED** [review]
dis is de best jov i si in yirs

File: anna-sala/challenges/to-do-list-redux/src/redux/reducers/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
+import tasksReducer from './tasksReducer';
+// import selectedTaskReducer from './selectedTaskReducer';
```
Comment (edalandete) [inline anna-sala/challenges/to-do-list-redux/src/redux/reducers/index.js:3]: comentaris

**Review by edalandete — APPROVED** [review]
molto benne puuuuuuuum

File: anna-sala/challenges/to-do-list-redux/public/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (shd8) [inline anna-sala/challenges/to-do-list-redux/public/index.html:27]: You should change the default title.

File: anna-sala/challenges/to-do-list-redux/src/components/Dashboard/index.jsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+          <input
+            id="new-task-name"
+            placeholder="New task Name"
+          />
+          <button type="button" className="add" onClick={() => clickAdd()}>ADD</button>
+        </label>
+      </form>
+      <ul className="to-do-list">
+        {tasks.map((task) => (
+          <li className="to-do-list__task">
```
Comment (shd8) [inline anna-sala/challenges/to-do-list-redux/src/components/Dashboard/index.jsx:37]: You should add a key to the fragment generated at every loop. In this example you could use:

<li key={task.id} >

Comment (shd8) [reply anna-sala/challenges/to-do-list-redux/src/components/Dashboard/index.jsx:37]: `<li key= {task.id} >`

**Review by shd8 — APPROVED** [review]
You have little changes pending, good work!

**Pull request merged**